pyItunes/__init__.py

```diff
@@ -1,4 +0,0 @@
-from pyItunes.XMLLibraryParser import XMLLibraryParser
```

Comment:
I felt a bit nervous about removing these from init, but couldn't figure out their purpose, and my project ran just fine without them. Confirm?